esi_leap/resource_objects/ironic_node.py

```diff
@@ -92,14 +92,6 @@ def expire_lease(self, lease):
         if state == "active":
             get_ironic_client().node.set_provision_state(self._uuid, "deleted")
 
-    def set_owner(self, owner_id):
```

Comment:
its probably fine but what did this piece of code do?

Reply:
It changed the owner of the node - which is pretty dangerous for several reasons.